pages/account/responsibility-sharing/dedicated-servers/guide.en-gb.md

```diff
@@ -0,0 +1,168 @@
+---
+title: RACI Dedicated servers
+slug: RACI-dedicated-server
+section : RACI
+excerpt: RACI between OVHcloud and the customer for Dedicated Servers
+---
+
+**Last updated December 1st, 2021**
+
+## Objective
+
+The RACI below details shared responsibilities between OVHcloud and the customer for Dedicated Servers services. This shared model can help relieve the customer’s operational burden.
+
+| Roles |
+| --- |
+| R: Is in charge of carrying out the process |
+| A: Guarantees the successful completion of the process |
+| C: Is consulted during the process |
+| I: Is informed of the results of the process |
+
+### 1. Before subscription
+
+#### 1.1. Specify service as needed
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Choose the location of dedicated servers | RA | I |
+| Size dedicated servers as needed | RA | I |
+| Choose options as needed | RA | I |
+
+### 2. Service availability
+
+#### 2.1. Install service
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Produce, transport, deliver and maintain physical machinery and buildings | CI | RA |
+| Install and configure the internal functional building blocks to the dedicated server that are required to keep it in operational conditions and secure conditions (firmware, BIOS, BMC, IPMI, etc.). |  | RA |
+| Buy and hold licences and usage rights for the OS used | RA |  |
+| Deploy initial network configuration to devices | I | RA |
+
+#### 2.2. Reversibility model
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
```

Comment:
Hello @GdMnt , this table is empty. Is it normal ?

Reply:
Same for the FR file

Reply:
yes